### ISSUE TITLE
Swap CD

### ISSUE THREAD

**Issue by dosboxonline** [caiiiycuk/js-dos.com#156]
Реализуйте функцию переключения дисков. Не всегда получается объеденить все образы в один чтобы обойти невозможность переключить образ. Иногда возможно изменить INI файлы и EXE вайлы HEX редактором, но например с игрой Harvester это не получиться поскольку имена файлов совпадают на разных дисках и они разные по размеру.

**Comment by caiiiycuk** [comment]
Я обычно делаю два отдельных бандла в этом случае... а вообще dosbox это поддерживает?

**Comment by dosboxonline** [comment]
Да, CTRL-F4 клавиши (монтирует следующий образ под выбранную букву диска). Только в браузере такая комбинация конечно же закрывает вкладку браузера, хотя может и работало бы. Может можно реализовать по клику на кнопку через скрипт вызов команд "unmount" и "mount" без выхода из игры?

**Comment by rwv** [comment]
I tried to use `ci.simulateKeyPress` to simulate `Ctrl+F4` but it didn't work.

**Comment by citee520** [comment]
Hi all:
about swap CD, I have implement it,
add call swapInNextDisk on server_add_key can successful.

**Comment by caiiiycuk** [comment]
@citee520 great!  Could you please submit a pull request?

**Comment by citee520** [comment]
OK~~I will try pull request.
